src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -241,15 +240,15 @@ class ReportActionItemMessageEdit extends React.Component {
                         onFocus={() => {
                             this.setState({isFocused: true});
                             ReportScrollManager.scrollToIndex({animated: true, index: this.props.index}, true);
-                            toggleReportActionComposeView(false, VirtualKeyboard.shouldAssumeIsOpen());
+                            toggleReportActionComposeView(false, this.props.isSmallScreenWidth);
```

Comment:
The main composer will hide when editing the message in decreased screen size on web/Desktop. Are we fine with that?

Reply:
Yeah, I am OK with that. I think it's closer to what we want anyway where those smaller windows would act closer to our mobile platforms. Thanks for asking!